transit_odp/publish/templates/publish/seasonal_services/index.html

```diff
@@ -71,25 +71,22 @@ <h1 class="govuk-heading-xl app-masthead__title">{% trans 'Seasonal service oper
             <div class="govuk-width-container">
                 <div class="govuk-grid-column-full custom_govuk_table_border">
                     {% if seasonal_services_counter %}
-                        {% render_table table %}
+                        <form method="post" enctype="multipart/form-data" action="{% url 'delete-seasonal-service' pk1=pk1 host hosts.publish %}?page={{ page_obj.number }}&">
```

Comment:
We probably do need the page number thing here though! This makes a lot of sense